nncf/torch/dynamic_graph/io_handling.py

```diff
@@ -329,7 +327,22 @@ def wrap_inputs(self, model_args: Tuple, model_kwargs: Dict) -> Tuple[Tuple, Dic
                 bound_model_params.apply_defaults()
 
             potential_tensor = bound_model_params.arguments[param_name]
+            if not isinstance(dummy_input, torch.Tensor):
+                if isinstance(potential_tensor, torch.Tensor):
+                    raise RuntimeError(
+                        f'Original model forward parameter "{param_name}" expected to not be a tensor,'
+                        " but the tensor type recieved. Please use a tensor type as an example input"
+                        f' for the "{param_name}" parameter_name'
+                    )
+                continue  # Did not expect a tensor at this parameter during graph building, shouldn't wrap now too
+
             if potential_tensor is not None:
+                if not isinstance(potential_tensor, torch.Tensor):
+                    nncf_logger.warning(
+                        f'Original model forward parameter "{param_name}" expected to be a tensor,'
+                        f" but {type(potential_tensor)} recieved."
+                    )
+                    continue
```

Comment:
From the user stand point, an user will see warning every model inference, right? I think this will piss off any user:)

I suggest to remove this warning.

Reply:
Done